sql/2024/third-parties/number_of_third_parties_by_rank.sql

```diff
@@ -0,0 +1,77 @@
+#standardSQL
+# Number of third-parties per websites by rank
+WITH requests AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    pageid AS page,
+    url
+  FROM
+    `httparchive.summary_requests.2024_06_01_*`
```

Comment:
I think this format also needs to be updated.

Reply:
@tunetheweb What shall I use for column `httparchive.summary_pages.2024_06_01.pageid` from `httparchive.all.pages` table and similarly for column `httparchive.summary_requests.2024_06_01.pageid` from `httparchive.all.requests`?

Reply:
This is fixed and can be marked as resolved.